plugin/src/main/java/org/opensearch/ml/action/connector/DeleteConnectorTransportAction.java

```diff
@@ -53,11 +56,13 @@ public DeleteConnectorTransportAction(
         TransportService transportService,
         ActionFilters actionFilters,
         Client client,
```

Comment:
Nit: client is no longer used and can be removed. Same comment for other transport action classes.

Reply:
It very much is still used.
```java
try (ThreadContext.StoredContext context = client.threadPool().getThreadContext().stashContext()) {
```
```java
client.threadPool().executor(GENERAL_THREAD_POOL)
```